File: BACKEND/backendDietiEstates/src/main/java/it/dietiestates/data/dto/PrenotazioneConInfo.java
package it.dietiestates.data.dto;

import it.dietiestates.data.Annuncio;
import it.dietiestates.data.Prenotazione;

public class PrenotazioneConInfo {
    private Prenotazione prenotazione;
    private Annuncio annuncio;

    public PrenotazioneConInfo(Prenotazione prenotazione, Annuncio annuncio) {
        setPrenotazione(prenotazione);
        setAnnuncio(annuncio);
    }

    public Prenotazione getPrenotazione() {
        return prenotazione;
    }

    public void setPrenotazione(Prenotazione prenotazione) {
        this.prenotazione = prenotazione;
    }

    public Annuncio getAnnuncio() {
        return annuncio;
    }

    public void setAnnuncio(Annuncio annuncio) {
        this.annuncio = annuncio;
    }
}

File: FRONTEND/DietiEstates/app/src/main/java/it/unina/dietiestates/data/dto/PrenotazioneConInfo.kt
package it.unina.dietiestates.data.dto

import it.unina.dietiestates.data.model.Annuncio
import it.unina.dietiestates.data.model.Prenotazione

data class PrenotazioneConInfo (
    val prenotazione: Prenotazione,
    val annuncio: Annuncio
)
